test/expressions/literals/literal-regexp-expression.js

```diff
@@ -25,5 +31,189 @@ suite('Parser', () => {
     testParseFailure('/(?:)/mm', 'Duplicate regular expression flag \'m\'');
     testParseFailure('/(?:)/yy', 'Duplicate regular expression flag \'y\'');
     testParseFailure('/(?:)/uu', 'Duplicate regular expression flag \'u\'');
+    testRegexAcceptFailure('/[/');
+    const regexToPass = [
```

Comment:
Can you write this as
```js
const regexToPass = String.raw`
/\bX/
/\BX/
/(?=t|v|X|.|$||)/u
[...]
`.split('\n');
```
instead? Then we don't have to deal with the annoying double escape thing.